src/locale.en.js

```diff
@@ -169,3 +172,19 @@ export const favorites = {
     must_log_in_detail: 'Only logged in users can save Favorite Sites. Log in and start saving all your favorites!'
 };
 
+export const counties = {
+    chaffee: 'Chaffee',
+    lake: 'Lake',
+    el_paso: 'El Paso'
+};
+
+export const forest = {
+  san_isabel: 'San Isabel',
+  pike: 'Pike'
+};
```

Comment:
Not using forests yet, but figured I'd start the object since we're already doing county and mvum office